docs/features/kubernetes/authentication.md

```diff
@@ -0,0 +1,35 @@
+---
+id: kubernetes-auth
+title: Kubernetes Authentication
+description: Authentication in Kubernetes plugin
+---
+
+The authentication process in kubernetes is basically separate from backstage auth, the
+providers are configured so your kubernetes plugin can locate and access the clusters you
+have access to, the providers currently available are categorized in server side auth and
+client side auth.
```

Comment:
Break up the run-on sentence, and a little rephrasing to avoid saying the word 'auth' so much
```suggestion
have access to. The providers currently available are divided into server side and
client side.
```